### PR TITLE
[FIX] Check order flags correctly (RLJS-279)

### PR DESCRIPTION
**Pull request by lumberj**

### CODE REVIEW

File: api/orders.js [hunk 1]
```diff
@@ -99,8 +99,8 @@ function getOrders(request, response, next) {
   function getParsedOrders(offers) {
     return _.reduce(offers, function(orders, off) {
       var sequence = off.seq;
-      var passive = off.flags === ripple.Remote.flags.offer.Passive;
-      var type = off.flags === ripple.Remote.flags.offer.Sell ? 'sell' : 'buy'
+      var passive = off.flags & ripple.Remote.flags.offer.Passive ? true : false;
```
Comment (clark800) [inline api/orders.js]: Could be slightly type-cleaner to do `off.flags & ripple.Remote.flags.offer.Passive !== 0`

Comment (lumberj) [inline api/orders.js]: :+1:

**Comment by lumberj** [comment]
@clark800 Updated

**Comment by clark800** [comment]
LGTM

**Pull request merged**